website/docs/configuring_clients.md

```diff
@@ -8,10 +8,342 @@ title: Configuring Clients
+            let currentDevice = UIDevice()
+
+            // -- optional config --
+            // set some 'useful' hardware information params
+            clientDevice.operatingSystem = currentDevice.systemName
+            clientDevice.osVersion = currentDevice.systemVersion
+            clientDevice.deviceType = currentDevice.model
+        },
+        success: {
+            print("UnifiedPush Server registration succeeded")
+        },
+        failure: { (error: Error!) in
+            print("failed to register, error: \(error.localizedDescription)")
+        }
+)
+```
+
+To make the code work, remember to replace the placeholder with the real values.
+
+### Handle the push notifications
+
+To handle the notification, in you `AppDelegate.swift` override the following method:
+
+```swift
+func application(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable: Any], fetchCompletionHandler: @escaping (UIBackgroundFetchResult) -> Void)
+```
+
+the `userInfo` parameters will contains all the information about the notifications. For further details, refer
+to the [apple website](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623013-application).
+
+### The Example application
+
+An example application can be found [here](https://github.com/aerogear/aerogear-ios-cookbook/tree/master/UnifiedPushHelloWorld).
+To run it, follow the instruction in the [README](https://github.com/aerogear/aerogear-ios-cookbook/blob/master/UnifiedPushHelloWorld/README.md) file.
+
+### Make the application backward compatible
+
+:::important
+These steps are required only if you created the project with _XCode 11_ and you need to support _iOS 12_ or older
+:::
+
+To make an application created with _XCode 11_ work with _iOS 12_ or older, few steps are necessary.
+
+#### Change the deployment target
+
+By default, when you create an application with _XCode 11_, it targets _iOS 13_. 
+
+To change the deployment target, open the project settings and go to the _General_ tab as shown below:
+
+>![CreateDemoApp](./assets/ios/ChangeTarget.png)
+
+#### Make the code backward compatible
+
+The _XCode 11_ application template, by default uses lot of things that are not available in _iOS 12_: to fix those issues, 
+we will have to mark them with the `@available` keyword.
+
+1. Change the `ContentView.swift` file so that its content is marked as available only on _iOS13_:
+
+```swift
+import SwiftUI
+
+@available(iOS 13.0, *)
+struct ContentView: View {
+    var body: some View {
+        Text("Hello, World!")
+    }
+}
+
+@available(iOS 13.0, *)
+struct ContentView_Previews: PreviewProvider {
+    static var previews: some View {
+        ContentView()
+    }
+}
+```
+
+2. Do the same for the `SceneDelegate.swift`
+
+```swift
+@available(iOS 13.0, *)
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+...
+```
+
+3. In the `AppDelegate` mark with the `@available` keyword each method referring to the `UIScene*` classes
```

Comment:
I think this line sounds more natural as : 

 In the `AppDelegate` mark each method referring to the `UIScene*` classes with the `@available` keyword.